src/Components/Admin/EnvironmentEditor/transitionModal.js

```diff
@@ -266,21 +286,35 @@ export default function TransitionModal({
                                     }}
                                     placeholder="Enter url to link (including http or https)"
                                 />
-                                {transition.link && (
-                                    <Button
-                                        className={classes.linkButton}
-                                        onClick={() =>
-                                            window.open(
-                                                transition.link,
-                                                "_blank"
-                                            )
-                                        }
-                                    >
-                                        Test URL
-                                    </Button>
-                                )}
+                                <Button
+                                    className={classes.linkButton}
+                                    onClick={() =>
+                                        window.open(transition.link, "_blank")
+                                    }
+                                    disabled={!transition.link}
+                                >
+                                    Test URL
+                                </Button>
+                            </div>
+                            <div>
+                                <p style={{ marginBottom: 0 }}>
+                                    Transition Text:
+                                </p>
+                                <TextField
+                                    value={transition.text}
+                                    onChange={(e) => handleTextChange(e, index)}
+                                    className={classes.textField}
+                                    required
+                                    variant="outlined"
+                                    inputProps={{
+                                        style: {
+                                            padding: 10,
+                                        },
+                                    }}
+                                    placeholder="Hint text"
```

Comment:
I think this should be transistions text or something?

Reply:
Yup you're right! Fixed.